thinkhazard_processing/models.py

```diff
@@ -54,6 +59,20 @@ class HazardSet(Base):
     # finally it is processed:
     processed = Column(Boolean, nullable=False, default=False)
 
+    hazardtype = relationship('HazardType', backref="hazardsets")
+
+    def path(self):
+        return os.path.join(settings['data_path'],
+                            'hazardsets',
+                            self.id)
```

Comment:
Is the `path` to hazard set really useful. Knowing the `path` to layers should be enough.